### PR TITLE
Fetch region from node info

### PR DESCRIPTION
**Pull request by ambiknai**

### CODE REVIEW

File: pkg/ibmcsidriver/ibm_csi_driver.go [hunk 1]
```diff
@@ -105,6 +108,13 @@ func (icDriver *IBMCSIDriver) SetupIBMCSIDriver(provider cloudProvider.CloudProv
 
 	icDriver.logger.Info("Successfully setup IBM CSI driver")
 
+	// Set up Region
+	regionMetadata, err := nodeMeta(os.Getenv("KUBE_NODE_NAME"), lgr)
```
Comment (sameshai) [inline pkg/ibmcsidriver/ibm_csi_driver.go:112]: can this nodeMeta be nil , please handle for nil condition

Comment (ambiknai) [reply pkg/ibmcsidriver/ibm_csi_driver.go:112]: https://github.com/IBM/ibm-csi-common/blob/master/pkg/metadata/metadata.go#L54
From this method, even if it is nil, error is returned. so we are handling the error before proceeding

**Review by sameshai — APPROVED** [review]
lgtm

File: deploy/kubernetes/driver/kubernetes/manifests/controller-server.yaml [hunk 2]
```diff
@@ -110,6 +110,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: KUBE_NODE_NAME
```
Comment (arahamad) [inline deploy/kubernetes/driver/kubernetes/manifests/controller-server.yaml:113]: This changes need to be done
in following repos as well

1- satellite
2- operator
3- template
4- addon

**Review by arahamad — APPROVED** [review]
lgtm, 

please make sure that we will do following changes and mention this changes in the release note

This changes need to be done
in following repos as well

1- satellite
2- operator
3- template
4- addon

**Review by arahamad — COMMENTED** [review]
/lgtm

**Pull request merged**